src/NuGet.Services.EndToEnd/Support/PushedPackagesFixture.cs

```diff
@@ -319,6 +319,11 @@ private async Task<List<PackageToPrepare>> InitializePackagesAsync(PackageType p
                     packageToPrepare = new PackageToPrepare(Package.Create(packageType, id, SemVer2PrerelVersion));
                     break;
 
+                case PackageType.SemVer2PrerelUnicodeId:
+                    // package Id contains unicode characters.
+                    packageToPrepare = new PackageToPrepare(Package.Create(packageType, id + "пакет包elsökning123", SemVer2PrerelVersion));

```

Comment:
I don't believe passing actual package Id from test adds any value, make code complicated, this this should be suffice. We have other unit test with more cases in NuGetJobs solution.